rpms.in.yaml

```diff
@@ -8,4 +8,5 @@ packages:
   - libstdc++-devel
   - clang
   - pkgconfig
-  - zeromq-devel
\ No newline at end of file
+  - zeromq-devel
+  - dnf
```

Comment:
Why do you need to install the dnf package inside your UBI-minimal production image just to be able to do `dnf install zeromq`? Just by installing the `dnf` package it results in an additional 25 packages installed in the image which will add to your image's footprint. Not only that, the image will undergo CVE scans for ALL packages installed inside and which you will need to promptly address. I strongly suggest sticking to `microdnf`.
```
...
Transaction Summary:
 Installing:       25 packages
```